[ON-18] 설문 참여 - 유형별 페이지 퍼블리싱

chldsbdud opened the pull request:
## 📌 주요 변경 사항
<!-- 이 PR에서 어떤 작업을 했는지 간단히 요약해주세요. -->

> 설문 참여 - 유형별 페이지 퍼블리싱
* 설문조사 시작 페이지
* 객관식
* 서술형, 단답형
* 평가형
* NPS
* 숫자형
* 날짜형



## 🔗 관련 이슈
<!-- 관련된 이슈 번호를 연결해주세요. -->
Linear 링크: https://linear.app/on-survey/issue/ON-18/설문-참여-디테일-페이지-퍼블리싱


 
## ✅ 리뷰 요청 사항 (Need Review)
<!-- 아래 옵션 중 하나 이상을 선택해주세요. -->

- [x] 🙂 **크게 우려되는 사항은 없어요.** 가볍게 리뷰 부탁드려요.  




## 🎨 스크린샷 (선택)
<!-- 작업한 내용의 뷰를 첨부해주세요. -->


https://github.com/user-attachments/assets/8d37e7db-7e0f-4762-a0ab-36f6c9dfb9b2

chldsbdud committed:
✨ Feat : 설문조사 시작 페이지 퍼블리싱

chldsbdud committed:
✨ Feat : 설문 참여 - 객관식 페이지

chldsbdud committed:
✨ Feat :설문 참여 - 서술형, 단답형 페이지

chldsbdud committed:
✨ Feat : 설문 참여 - 평가형 페이지

chldsbdud committed:
♻️ Refactor : rating, NPS  불필요한 로직 및 코드 삭제

chldsbdud committed:
✨ Feat :  설문참여 - 숫자형 페이지

chldsbdud committed:
✨ Feat : 설문 참여 - 날짜형 페이지

chldsbdud commented:
작업 내용이 조금 많습니다..!! 시간 될 때 확인 부탁드리겠습니다

78-artilleryman commented on src/pages/survey/Number.tsx:
```diff
@@ -0,0 +1,101 @@
+							setValue(val);
+						}
+					}
+				}}
```
유효성 검사 로직은 따로 함수로 분리하면 좋을 거 같아요
그리고 이번에 우리가 리엑트훅폼 쓰기로 했으니 그걸 사용해서 상태와 유효성 검사까지 할 수 있도록 바꿔보면 어떨까요?

78-artilleryman commented on src/pages/survey/SingleChoice.tsx:
```diff
@@ -0,0 +1,124 @@
+								<Checkbox.Line
+									checked={selectedChoiceId === choice.id}
+									aria-hidden={true}
+									style={{ pointerEvents: "none" }}
```
여기 pointerEvents 스타일 빼줘야해요

78-artilleryman reviewed (dismissed):
고생하셨네요
리뷰 확인 부탁드립니다

chldsbdud committed:
♻️ Refactor : 숫자 입력값  유효성 검사 함수 분리

chldsbdud committed:
🔥 Del : 불필요 스타일 코드 제거

chldsbdud committed:
🐛 App.tsx 충돌 해결

chldsbdud committed:
🔀 Resolve: main 브랜치와 App.tsx 충돌 해결

chldsbdud committed:
WIP: main 머지 전 임시 커밋

chldsbdud committed:
🔀 Merge: main 브랜치 머지 및 충돌 해결

- App.tsx: 설문 참여 라우트 유지하면서 main 브랜치 구조 반영
- tsconfig.app.tsbuildinfo: 삭제 (빌드 결과물)

78-artilleryman reviewed (approved):
좋습니다 머지하시죠! 🔥
